src/platformio.ini

```diff
@@ -336,8 +336,8 @@ src_filter = ${env:DIY_900_RX_ESP8285_SX127x_via_UART.src_filter}
 upload_speed = 74880
 upload_protocol = custom
 upload_command =
-	python python/runpython.py $PYTHONEXE python/BFinitPassthrough.py 74880
-	python $PROJECT_PACKAGES_DIR/framework-arduinoespressif32/tools/esptool.py --no-stub -b 74880 -c esp8266 --before no_reset --after soft_reset write_flash 0x0000 $SOURCE
+	python $PROJECT_DIR/python/BFinitPassthrough.py 74880
```

Comment:
replace `74880` by `$UPLOAD_SPEED` to avoid unnecessary copy of the same value. 
will be easy to try different values :)

Reply:
Bloddy hell... I went back and fixed it in the esptool part to use `$UPLOAD_SPEED` but totally missed it for init passthrough apparently?

Reply:
wait. it's completly gone... gonna fix it